app/src/main/java/de/s3xy/architecturesample/di/ApplicationModule.java

```diff
@@ -13,16 +16,23 @@
 @Module
 public class ApplicationModule {
 
-    private final Application mApplication;
+    private final Context mAppContext;
 
     public ApplicationModule(Application application) {
```

Comment:
If we do not need Application anymore and just get the context would it be better to hand in context directly rather than getting it from the application?